### PR TITLE
Open sourcing hparams 2

### PR DESCRIPTION
**Pull request by erzel**
Minor tweaks:
1. Added "licenses()" function calls to the hparams BUILD files
2. Fixed 'legacy' targets dependencies.
3. Added a visibility whitelist for the tensorboard_webcomponent_library version of hparams (only for internal google usage).

### CODE REVIEW

**Comment by erzel** [comment]
Hi @stephanwlee 

As I anticipated, there are a few more changes. But these ones are much more minor.
Also, I just remembered I forgot to address your comment on the "100ms" I'm using in debounce. I did not choose that number in any "principled" way. Can you explain a little more about your idea to use requestAnimationFrame?

Thanks,

 Erez.

File: tensorboard/plugins/hparams/tf_hparams_main/BUILD [hunk 1]
```diff
@@ -25,14 +27,13 @@ tf_web_library(
 tensorboard_webcomponent_library(
     name = "legacy",
     srcs = [":tf_hparams_main"],
-    visibility = [":tf_hparams_main_users"],
+    visibility = ["//tensorboard/plugins/hparams/external_users:whitelist"],
```
Comment (stephanwlee) [inline tensorboard/plugins/hparams/tf_hparams_main/BUILD]: This does not appear to exist. Can you double check please?

**Comment by erzel** [comment]
Thanks Stephan; yes, the idea is that this would only exist in Google3. The
tensorboard_webcomponent_library macro just evaluates to nothing in
open-source, I believe.

 Erez./

**Comment by erzel** [comment]
Thanks Stephan!

File: tensorboard/plugins/hparams/tf_hparams_main/BUILD (hunk 1 above)
Comment (stephanwlee) [reply tensorboard/plugins/hparams/tf_hparams_main/BUILD]: I am not so thrilled about having a ghost target like this even if it not used in GitHub. Can we do something about it? (Perhaps a shim target that gets replaced internally?)

**Pull request merged**